composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "ezsystems/ezpublish-kernel": "^6.13.4@dev || ^7.3@dev",
+        "ezsystems/ezpublish-kernel": "^6.13.4@dev || ^7.5@dev",
```

Comment:
Will it work for 2.4 branch? Ezplatform 2.4 uses both:
`"ezsystems/ezpublish-kernel": "~7.4.0@dev",`
`"ezsystems/ezplatform-http-cache": "~0.9.0@dev",`
(https://github.com/ezsystems/ezplatform/blob/2.4/composer.json#L36)
which will clash with this requirement if I'm not mistaken

Reply:
Good catch Marek. This code can not crash 2.4 version. Is this doable?

Reply:
it will not afaik, if it does, then we also have same issue on 1.13

Reply:
Actually, it should work fine without `composer.json` modification (signal is not emitted => slot will not be executed). We will check this during QA.

Reply:
Yeah, it's only solr having issues with this due to the use of abstract search slots, which is in retrospective not a good idea.